test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -107,6 +107,19 @@ describe('Dropdown Component', () => {
     dropdownMenuIsClosed()
   })
 
+  it('blurs the Dropdown node on close', () => {
+    wrapperMount(<Dropdown options={options} selection defaultOpen />)
+
+    dropdownMenuIsOpen()
+    wrapper.simulate('click')
+    dropdownMenuIsClosed()
+
+    const instance = wrapper.instance()
+    sandbox.spy(instance._dropdown, 'blur')
```

Comment:
Thanks for your help, makes sense now 😄 
I changed
```js
instance._dropdown.blur
  .should.have.been.calledOnce()
```
to
```js
instance._dropdown.blur
  .should.have.been.called()
```

because blur() was called twice. Is that alright?